### PR TITLE
Added all other ecommerce pages with wishlist and cart functionality plus filter.

### PR DESCRIPTION
**Pull request by Hamza-777**
Added all remaining pages to my ecomm website please review and provide feedback.
Live: https://aquamarine-liger-797ab3.netlify.app/

### CODE REVIEW

**Review by md-amir-gauhar — APPROVED** [review]
-> Clean UI
looks good to merge

File: src/Components/Pages/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                        <label for="password">Password</label>
+                        <div className="pass">
+                            <input type={type} name="password" placeholder="Enter your password" className="password" value={password} onChange={changeHandler} />
+                            <i className={`far ${eye} togglePassword`} onClick={toggleView}></i>
+                        </div>
+                    </div>
+                    <div className="form-item flex flex-col align-start misc-inputs">
+                        <div className="remember">
+                            <input type="checkbox" name="checked" id="remember" checked={checked} onChange={changeHandler} />
+                            <label for="password small">Remember Me</label>
```
Comment (hsnice16) [inline src/Components/Pages/Login.jsx:50]: instead of `for`, in jsx we use `htmlFor`. I was checking your UI, and found that it is not working.

**Review by hsnice16 — APPROVED** [review]
The code is good. But I would suggest to use some kind of formatter, like prettier, to format code well.

Right now, it is hard to read the code easily.

File: src/Components/Pages/Login.jsx (hunk 1 above)
Comment (Hamza-777) [reply src/Components/Pages/Login.jsx:50]: Okay, will do it

**Pull request merged**